samples/pet-store/cellery/pet-be.bal

```diff
@@ -0,0 +1,122 @@
+
+//   Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+// Cellery file for building backend Pet-store sample cell.
+
+import ballerina/io;
+import celleryio/cellery;
+
+// Pet-store Components
+// Orders Component
+cellery:Component ordersComponent = {
+    name: "orders",
+    source: {
+        image: "celleryio/samples-pet-store-orders"
+    },
+    ingresses: {
+        orders: new cellery:HTTPIngress(80,
+            "orders-svc",
+            [
+                {
+                    path: "/orders",
+                    method: "GET"
+                }
+            ]
+        )
+    }
+};
+
+// Customers Component
+cellery:Component customersComponent = {
+    name: "customers",
+    source: {
+        image: "celleryio/samples-pet-store-customers"
+    },
+    ingresses: {
+        customers: new cellery:HTTPIngress(80,
+            "customers-svc",
+            [
+                {
+                    path: "/customers",
+                    method: "GET"
+                }
+            ]
+        )
+    }
+};
+
+// Catalog Component
+cellery:Component catalogComponent = {
+    name: "catalog",
+    source: {
+        image: "celleryio/samples-pet-store-catalog"
+    },
+    ingresses: {
+        catalog: new cellery:HTTPIngress(80,
+            "catalog-svc",
+            [
+                {
+                    path: "/catalog",
+                    method: "GET"
+                }
+            ]
+        )
+    }
+};
+
+// Controller Component
+cellery:Component controllerComponent = {
+    name: "controller",
+    source: {
+        image: "celleryio/samples-pet-store-controller"
+    },
+    ingresses: {
+        controller: new cellery:HTTPIngress(
+                      80,
+                      "controller",
+                      "./resources/pet-store.swagger.json"
```

Comment:
Point to existing swagger file in https://github.com/wso2-cellery/sdk/blob/master/samples/pet-store/components/controller/resources/pet-store.swagger.json